source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -85,6 +86,14 @@ following text:
       ...
    }
 
+.. note:: ChangeStreamIterable and ChangeStreamDocument Types
+   
+   The driver stores change stream events in a variable of type
+   ``ChangeStreamIterable``. In the preceding example, the driver stored
+   individual change stream events as ``ChangeStreamDocument`` objects,
+   since we specified that the driver should populate the ``ChangeStreamIterable``
+   with ``Document`` types.
```

Comment:
[Suggestion]

A couple of things here. First, the sentence is a little long, so I would recommend breaking it up. Also, I would put the action that we did first, to put the focus on that, and then have the effect later. Something like the following:

```suggestion
   ``ChangeStreamIterable``. In the preceding example, we specified the driver should
   populate the ``ChangeStreamIterable`` with ``Document`` types. As a result, the driver
   stores the individual change stream events as ``ChangeStreamDocument`` objects.
```